bitnami/postgresql/templates/primary/servicemonitor.yaml

```diff
@@ -8,7 +8,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ include "postgresql.v1.primary.fullname" . }}
-  namespace: {{ default .Release.Namespace .Values.metrics.serviceMonitor.namespace | quote }}
+  namespace: {{ coalesce .Values.metrics.serviceMonitor.namespace (include "common.names.namespace" .) | quote }}
```

Comment:
Why switching default with coalesce + switching order?

Reply:
I decided to use coalesce because it is already used in postgresql-ha chart and I wanted to reduce the differences between these charts. If you want I could change it back and use default again.

Reply:
I also like the fact that the value with the highest precedence is on the left side.

Reply:
All right, let's switch to coalesce if that help reducing differences.